cdap-api/src/main/java/co/cask/cdap/internal/Objects.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2007 The Guava Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package co.cask.cdap.internal;
+
+import java.util.Arrays;
+import javax.annotation.Nullable;
+
+/**
+ * Helper functions that can operate on any {@code Object}.
```

Comment:
Better not to copy from guava. This class will become not useful soon (Java 7 has java.uti.Objects).